docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,155 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
+## Defining a HasMany Relation
+
+The first relation type we have available is the `hasMany` relation which
```

Comment:
We should probably reword this sentence so that we don't give precedence (ordering) to the relations we support

Reply:
What Diana has said